package.json

```
@@ -7,7 +7,7 @@
   "scripts": {
     "lint": "eslint . --ext .js",
     "test": "npm run lint && npm run test:server",
```

```suggestion
    "test": "npm run lint && npm run test:server && npm run test:browser",
```
Now do we want this command to include everything?